q2_feature_classifier/_skl.py

```diff
@@ -75,13 +74,13 @@ def _predict_chunk_with_conf(pipeline, separator, confidence, chunk):
     split_classes = [c.split(separator) for c in pipeline.classes_]
     for seq_id, taxon, class_probs in zip(seq_ids, y, prob_pos):
         taxon = taxon.split(separator)
-        classes = zip(deepcopy(split_classes), class_probs)
+        classes = zip(split_classes, class_probs)
```

Comment:
Suggest changing this to
`classes = list(zip(split_classes, class_probs))`
and deleting line 83.